be/src/storage/kv_store.cpp

```diff
@@ -242,17 +243,39 @@ Status KVStore::iterate(ColumnFamilyIndex column_family_index, const std::string
     } else {
         it->Seek(prefix);
     }
-    for (; it->Valid(); it->Next()) {
-        if (!prefix.empty()) {
-            if (!it->key().starts_with(prefix)) {
-                return Status::OK();
+    // if limit time is less than zero, it means no limit
```

Comment:
<= 0?

Reply:
> <= 0?

done, change in https://github.com/StarRocks/starrocks/pull/19347/commits/ab09d917afdd3617d180b83cb64bf9ae98ec8cd2